### ISSUE TITLE
Install not working

### ISSUE THREAD

**Issue by rugene76** [JoshuaMulliken/ha-wyzeapi#32]
I'm new to HA.
I've installed the latest version of HA (hassio). In general everything seems to work regarding the devices I have that are officially supported, eg., wemo
However, when I attempt to install wysezpi, the instructions don't match the latest UI in regards to the repository.
I've attempted to do a manual install which I think is working - at least I'm seeing my wyze lights/switches..One thing to keep in mind: You must turn off two factor login for the Wyze app

so in short I am lost regarding the installation of custom repository
"Supervisor - Add on store" page.

**Comment by rugene76** [comment]
here's the log I found:
Error during setup of component wyzeapi

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/setup.py", line 174, in _async_setup_component
    component.setup, hass, processed_config  # type: ignore
  File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/wyzeapi/__init__.py", line 38, in setup
    config[DOMAIN].get(CONF_PASSWORD))
  File "/config/custom_components/wyzeapi/wyzeapi/wyzeapi.py", line 20, in __init__
    self.initialize()
  File "/config/custom_components/wyzeapi/wyzeapi/wyzeapi.py", line 26, in initialize
    self._access_token = self.login(self._user_name, self._password, self._device_id)
  File "/config/custom_components/wyzeapi/wyzeapi/wyzeapi.py", line 51, in login
    data = self._request_man.do_single_threaded_request(url, payload)
  File "/config/custom_components/wyzeapi/wyzeapi/wyzeapi_request_manager.py", line 82, in do_single_threaded_request
    self.request_worker(request)
  File "/config/custom_components/wyzeapi/wyzeapi/wyzeapi_request_manager.py", line 66, in request_worker
    raise WyzeApiError(data['msg'])
custom_components.wyzeapi.wyzeapi.wyzeapi_exceptions.WyzeApiError: ParameterError

**Comment by JoshuaMulliken** [comment]
Do you have Home Assistant Community Store (HACS) installed?

**Comment by rugene76** [comment]
Yes, I have HACS installed. I'm not seeing anything that would help me there. Is there something?
It seems what I need to do is create a fake "dimmable light" as an entity and then send event triggers to it to kick off the automation.  That way I could set the position of a blind using the dimmer interface and resultant parameter data. but I don't see how to do create a fake entity.

**Comment by rugene76** [comment]
Yes I do and I have finally got the sensors to work - not sure why but they work. Yea!

**Comment by JoshuaMulliken** [comment]
Great!
